upt: format routes

linbuxiao opened the pull request:
NOROUTE

## 说明
### 完成部分
1. 去除了`link-module-alias`的依赖
2. 重新优化了routes部分的`cjs-esm`的转化
3. `pakeage.json`新增了`imports`，用于替换`alias`

目前首页，`v2-routes`正常加载启动

### 剩余问题
1. `vercel`部署失效？
2. `old-routes`部分由于`alias`路径替换不彻底以及`esm`标准下的`.js`后缀问题，无法启动，需要进行测试排查。

tomyangsh committed:
fix(route)(zimuku) 字幕库新域名 (#8291)

james-tindal committed:
fix(core)(docs): Can't commit some changes in docs folder (#8348)

UniQMG522 committed:
fix(route)(firefox): route for firefox addons (#7239)

Co-authored-by: NeverBehave <gayhub@never.pet>

wyq977 committed:
fix(route)(ptwxz): ptwxz add fulltext (#8331)

Co-authored-by: auto-bot-ty <75887908+auto-bot-ty@users.noreply.github.com>
Co-authored-by: NeverBehave <gayhub@never.pet>

Commit:
feat(route): add 企鹅直播直播间开播 (#8345)

actions-user committed:
style: auto format

Commit:
feat(route): add Focus Taiwan (#8338)

Commit:
feat(route): add 2047 (#8302)

auto-bot-ty committed:
fix(route): 小黑盒 - 用户动态 (#7987)

Co-authored-by: SettingDust <settingdust@gmail.com>

auto-bot-ty committed:
fix(route): 動畫瘋最後更新 (#8018)

aidistan committed:
fix(route): 生物探索 (#8098)

yindaheng98 committed:
feat(route): Twitter用户时间线和用户媒体时间线不再需要开发者账户 (#7739)

actions-user committed:
style: auto format

Discreater committed:
fix(route): minecraft forge mod 使用新 api (#7807)

NeverBehave reviewed (changes requested):
1. 按照我之前给你的样例， getConfig 应当被直接导出，类似的名字修订同理，具体名字导出避免重命名

NeverBehave commented:
2. 如果可以，不要限定js，手动加入后缀，为后续ts做准备

NeverBehave commented:
3. 不要使用var，默认导出名字你可以思考一下，default不一定是最优解法，比如路由导出就可以是 
```
import { someroute1, someroute2 } from xxx
```

因为我们按需载入是按照模组进行的，文件本身载入是没有必要懒加载，我们中间件会处理

denis-ya committed:
feat(route): add fisher spb news (#8343)

* feat(route): add fisher spb news

* Apply suggestions from code review

Co-authored-by: Tony <TonyRL@users.noreply.github.com>

* templates art for html render

Co-authored-by: Tony <TonyRL@users.noreply.github.com>

actions-user committed:
style: auto format

DIYgod committed:
docs: update sponsor

DIYgod committed:
feat: add hotukdeals

DIYgod committed:
feat: hotukdeals radar

DIYgod committed:
feat: add PS5 stock UK - The Independent

shizunge committed:
fix(route): allow preLoginFlow of instagram to fail (#8340)

Based on discussion in instragram-private-api repo, we do not need
this function for login.

See:
https://github.com/dilame/instagram-private-api/issues/1411
https://github.com/dilame/instagram-private-api/issues/1417

actions-user committed:
style: auto format

DIYgod committed:
feat: pixiv PIXIV_IMG_PROXY

NeverBehave commented:
ESM Test Checklist:

- [ ] Root page `\` works
- [ ] Randomly choose two route in `v1` and works
	- [ ] Record your test route here
	- [ ] Preferred more deps e.g. puppetter
- [ ] Random choose two v2 route and test
  - Note: v2 route may need to have further changes, if this part doesn't work, leave a message: we may need to revise our standard
  - !!Potential breaking changes
- [ ] Test case
   - [ ] It is possible that some test case may need to rewrite, leave a note for all failing test case

The main idea is to **ensure all core functions are working**. If there are some incompatibility, let the community to open issue.

linbuxiao committed:
ref: will install

linbuxiao committed:
ref: not yet

linbuxiao committed:
ref: core running

linbuxiao committed:
ref: v2-router finished

NeverBehave committed:
fix async
